README.md

```diff
@@ -13,28 +13,51 @@ Simple app displaying a leaderboard of Pokémon Go trainers.
 - [ ] Advanced profile page, with charts
 - [ ] Registration flow where the player has to tell his trainer name
 
-### Requirements
+### Requirements:
 
 - node > 14
 - Golbat
 
-## Installation
+### Installation:
 
 ```shell
 npm ci
 cp config/env/config.example config/env/production.json
 ```
 
+### Configuration:
 Then edit the file `config/env/production.json` to put your configuration.
 
-## Running the application
+### Discord Auth
+* Create a [discord bot](https://discord.com/developers) and fill in the config options below. All variable require a value. 
```

Comment:
> All variable require a value
I've now fixed an issue with that. Now, only the `clientId` and the `clientSecret` are required. The `botToken` is required if the `guildId` or the `roleId` are filled. If not used, the `guildId`/`roleId` should be empty arrays.

Reply:
I have not updated this in the readme

Reply:
guildId should be available with oauth alone, but those are small details

Reply:
Here the guildId is a filter. It's to be sure that the person login in through oauth2 is a member of your server.